Formula/ansible.rb

```diff
@@ -59,40 +60,24 @@ class Ansible < Formula
   end
   ### end
 
-  resource "Babel" do
```

Comment:
Heh, no big deal. I was stressed about something and sorting this stuff out was the most sensible distraction I could've plucked out of the air. Didn't bother me doing it. Before `poet` I used to do it all by hand, so, times have changed for the better on resource creation/sorting.